fala/apps/adviser/tests/test_adviser_extras.py

```diff
@@ -15,3 +15,8 @@ def test_returns_the_value_of_a_query_parameter_when_a_prop_is_given(self):
     def test_returns_an_empty_list_when_a_prop_is_given_but_does_not_exist(self):
         result = adviser_extras.query_to_dict("irrelevant", "http://localhost:8000/?postcode=SW1A&page=3", "potato")
         self.assertEqual([], result)
+
+    def test_google_map_params(self):
```

Comment:
I'd personally like to see more tests for different case scenarios instead of just a happy one. 

- Test without address
- Test without postcode
- Test that the function handles None values for the location parameter gracefully. (or Unexpected values) how does it handle this?

Reply:
It doesn't, and I couldn't find any reason for the code to be more complicated than it currently is.

Reply:
But for defence purposes, we shouldn't always assume that the data is always going to be fine right though surely?

Reply:
I think Pat's right here. At the very least, I don't want future devs having to come here and work out they need to write extra tests when they want to change laalaa because we haven't been a little bit defensive